### PR TITLE
fix: only consider unique eventTriggers

### LINKED ISSUE #13079: Create and update webhook api accepts repeated eventTriggers
### Issue Summary

Create and update webhook api accepts repeated eventTriggers. For example webhook can be created/updated with eventTriggers value as,

```
[
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED",
"BOOKING_CREATED"
]
```
Issue 7-1 1
Issue 7-1 2

### PR DESCRIPTION
**Pull request by riddhesh-mahajan**
## What does this PR do?

This PR removed repeated eventTriggers in create/update webhook api 

Fixes #13079 

Issue 7-1 sol1
Issue 7-1 sol2

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/api/pages/api/webhooks/[id]/_patch.ts [hunk 1]
```diff
@@ -87,6 +88,11 @@ export async function patchHandler(req: NextApiRequest) {
     args.data.userId = bodyUserId;
   }
 
+  if (args.data.eventTriggers) {
+    const uniqueEventTriggers = Array.from(new Set(args.data.eventTriggers as WebhookTriggerEvents[]));
+    args.data.eventTriggers = uniqueEventTriggers;
+  }
+
```
Comment (alishaz-polymath) [inline apps/api/pages/api/webhooks/[id]/_patch.ts:95]:
```suggestion
if (args.data.eventTriggers) {
  args.data.eventTriggers = [...new Set(args.data.eventTriggers)];
}
```

We don't need the intermediary const uniqueEventTriggers here, it should be doable without the need of it. If you see any type errors, feel free to fix that within the suggestion 🙏

File: apps/api/pages/api/webhooks/_post.ts [hunk 2]
```diff
@@ -87,6 +88,11 @@ async function postHandler(req: NextApiRequest) {
     args.data.userId = bodyUserId;
   }
 
+  if (args.data.eventTriggers) {
+    const uniqueEventTriggers = Array.from(new Set(args.data.eventTriggers as WebhookTriggerEvents[]));
+    args.data.eventTriggers = uniqueEventTriggers;
+  }
+
```
Comment (alishaz-polymath) [inline apps/api/pages/api/webhooks/_post.ts:95]:
```suggestion
if (args.data.eventTriggers) {
  args.data.eventTriggers = [...new Set(args.data.eventTriggers)];
}
```

Same as above

**Review by alishaz-polymath — CHANGES_REQUESTED** [review]
Thank you for the PR, great job 🎉 
Left minor code improvements, and it should be mergeable as soon as they're sorted. 🙌

File: apps/api/pages/api/webhooks/[id]/_patch.ts (hunk 1 above)
Comment (riddhesh-mahajan) [reply apps/api/pages/api/webhooks/[id]/_patch.ts:95]: Looks fine 👍

File: apps/api/pages/api/webhooks/_post.ts (hunk 2 above)
Comment (riddhesh-mahajan) [reply apps/api/pages/api/webhooks/_post.ts:95]: Looks fine 👍

**Comment by alishaz-polymath** [comment]
@riddhesh-mahajan you haven't updated the code yet 🤔

**Comment by riddhesh-mahajan** [comment]
> @riddhesh-mahajan you haven't updated the code yet 🤔 Am I missing something 😅

😂 Just pushed the update

**Review by alishaz-polymath — APPROVED** [review]
LGTM 🚀

**Pull request merged**